packages/xstate-graph/src/graph.ts

```diff
@@ -319,3 +322,44 @@ export function getSimplePathsAsArray<
   const result = getSimplePaths(machine, options);
   return keys(result).map((key) => result[key]);
 }
+
+export function toGraph(stateNode: StateNode): DigraphNode {
```

Comment:
If this returns `DigraphNode` shouldnt we call it `toDigraph`?

Reply:
I'll rename to `toDirectedGraph`